production/ksonnet/README.md

```diff
@@ -14,9 +14,12 @@ In your config repo, if you don't yet have the directory structure of Tanka set
 
 ```bash
 # create a directory (any name works)
-$ mkdir config && cd config/
+$ mkdir promtail && cd promtail/
```

Comment:
This is not really exactly correct, you would create a directory (tanka app) for ALL of your configs and not a separate one for promtail, which means `config` is actually pretty fine